### PR TITLE
Adding soft delete fieds to DirectDepositAccount

### PR DESCRIPTION
**Pull request by antonyw89**

### CODE REVIEW

File: source/includes/changelog/changelog.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 This is where you can find a list of changes to the Bullhorn REST API. This change log was initiated with the 2024.5 release. Prior changes are not listed by release.
 
+## August 2024 Release (2024.9)
+
+| ADDED ```|``` [DirectDepositAccount isDeleted, deletedByUserID field](http://bullhorn.github.io/rest-api-docs/entityref.html#pay-and-bill-directdepositaccount) |
```
Comment (Allblues) [inline source/includes/changelog/changelog.md:7]: Change field to fields

File: source/includes/entityref/paybill/_directdepositaccount.md [hunk 2]
```diff
@@ -113,14 +113,28 @@ Entity that represents a candidate's direct deposit account record.
             <td>X</td>
             <td></td>
         </tr>
+        <tr class="even">
```
Comment (Allblues) [inline source/includes/entityref/paybill/_directdepositaccount.md:116]: For now, match other entities like candidate, which has:

Indicates whether this record is marked as deleted in the Bullhorn system.

File: source/includes/entityref/paybill/_directdepositaccount.md [hunk 3]
```diff
@@ -113,14 +113,28 @@ Entity that represents a candidate's direct deposit account record.
...
+            <td>isDeleted</td>
+            <td>Boolean</td>
+            <td>Soft deleted flag.</td>
+            <td>X</td>
+            <td></td>
+        </tr>
+        <tr class="odd">
+            <td>deletedByUserID</td>
+            <td>Integer</td>
+            <td>User that soft deleted this Direct Deposit Account.</td>
```
Comment (Allblues) [inline source/includes/entityref/paybill/_directdepositaccount.md:126]: Let's go with something like:
User who deleted this DirectDepositAccount.

**Review by Allblues — CHANGES_REQUESTED** [review]

File: source/includes/changelog/changelog.md (hunk 1 above)
Comment (antonyw89) [reply source/includes/changelog/changelog.md]: Fixed

File: source/includes/entityref/paybill/_directdepositaccount.md (hunk 2 above)
Comment (antonyw89) [reply source/includes/entityref/paybill/_directdepositaccount.md:116]: Changed description

File: source/includes/entityref/paybill/_directdepositaccount.md (hunk 3 above)
Comment (antonyw89) [reply source/includes/entityref/paybill/_directdepositaccount.md]: Changed description

**Pull request merged**